3-typescript/2-custom-utility-types-and-generics/2-custom-utilities.ts

```diff
@@ -88,8 +115,16 @@
  */
 
 // Add here your solution
-
+type MyReturnType<T> = T extends (...args: any) => infer R ? R : any;
```

Comment:
1. Accept any number of arguments.
2. It's usually better to return `never` as an empty type.

```suggestion
type MyReturnType<T> = T extends (...args: any[]) => infer R ? R : never;
```